src/csharp/Microsoft.Spark/Broadcast.cs

```diff
@@ -37,7 +37,7 @@ internal Broadcast(SparkContext sc, T value)
             _bid = (long)_jvmObject.Invoke("id");
         }
 
-        JvmObjectReference IJvmObjectReferenceProvider.Reference => _jvmObject;
+        public JvmObjectReference Reference => _jvmObject;
```

Comment:
`_jvmObject` needs to be handled differently so that we don't serialize it, I assume?

Reply:
Yes, I didn't want to mess with this it's currently working and not following any of the other patterns.